### DISCUSSION TITLE
Error updating 0.15.0 Dev to 0.15.5 stable

### DISCUSSION THREAD

**Discussion by tnemrap** [inventree/InvenTree#7681] [Q&A]
When running invoke update i get this error:

Synchronizing apps without migrations:
  Creating tables..
    Running deferred SQL..
Running migrations:
  No migrations to apply.
InvenTree database migrations completed!
Updating frontend...
Compiling frontend code...
Installing frontend dependencies
00h00m00s 0/0: : ERROR: [Errno 2] No such file or directory: 'install'
(env) inventree@astrid:~/src$ 

I can run invoke server and access the application.

**Comment by SchrodingersGat** [comment]
If you are running from the `./src/` directory, try moving up one directory and running the command from there.
